index.bs

```diff
@@ -743,9 +742,18 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
                     <code>config.audio.contentType</code>.
                   </li>
                   <li>
-                    Set the {{EME/robustness}} attribute to
+                    Let <var>robustness</var> be a string <code>DOMString</a>
+                    initialized to the empty string (<code>""</code>).
```

Comment:
After sitting on this a bit more I think the approach is fine. 

We might simplify what you have here even further with something like:
 "if audioRobustness is present, set EME robustness attribute to  config.keySystemConfig.audioRobustness"

And simply do nothing if it audioRobustness is _not_ present. The default value from the EME config is "", so doing nothing should cause that default to take hold. 

Maybe thats too subtle? Open to either route.

Reply:
I don't have a strong opinion. In general, I find having clear defaults better than relying on side effects but if you feel strongly, I'm happy to update the PR.

Reply:
> "be a string <code>DOMString</a>"
Do you mean to say 'string' twice - just DOMString?

> Set the robustness attribute to robustness.
How about this change in wording style?

If config.keySystemConfiguration.videoRobustness is present, set robustness to config.keySystemConfiguration.videoRobustness. Otherwise, set robustness to "".

Reply:
I fixed "string DOMString".

For the wording, the way I see it is that I prefer:
```
let robustness = "";
if (config.keySystemConfiguration.videoRobustness)
  robustness = config.keySystemConfiguration.videoRobustness;
```
over:
```
let robustness = config.keySystemConfiguration.videoRobustness
                               ? config.keySystemConfiguration.videoRobustness
                               : "";
```
(reason is that I prefer to have the default clear if there is one)

It's pretty much the same at the end so if you think it's clearer, I'm happy to apply the change.